SpatialGDK/Documentation/known-issues.md

```diff
@@ -19,3 +19,4 @@ Known issue = any major user-facing bug or lack of user-facing feature that:
 | Seamless Travel is not supported | 2019-01-22 | [UNR-897](https://improbableio.atlassian.net/browse/UNR-897) | Disable Seamless travel |  
 | Gameplay Ability System is not fully supported | 2019-02-01 | | Use the workarounds detailed on the [Gameplay Ability System]({{urlRoot}}/content/ability-system) reference page. |  
 | Opening the Editor while you are uploading or launching a cloud deployment disables the GDK Toolbar **Start** button, and Selecting **Stop** on the GDK Toolbar stops the running cloud deployment. | 2019-02-18 | [UNR-1006](https://improbableio.atlassian.net/browse/UNR-1006) | Wait for the cloud launch to finish, or launch the cloud deployment whilst the Editor is already open. |
+| Blueprint RPCs with "by reference" arguments are not serialized correctly | 2019-02-18 | [UNR-1011](https://improbableio.atlassian.net/browse/UNR-1011) | Change any "by reference" arguments to "by value" (default). |
```

Comment:
>not serialized correctly 
When and by what/who?

Reply:
When sending an RPC through Spatial. By server or client depending on the type of RPC.
I'm not sure this adds information so I didn't include it.

Reply:
Ace, thanks. In that case: 

```suggestion
| Blueprint RPCs with "by reference" arguments are not serialised correctly when they are sent through SpatialOS. | 2019-02-18 | [UNR-1011](https://improbableio.atlassian.net/browse/UNR-1011) | Change any "by reference" arguments to "by value" (default). |
```

Reply:
(serialised with an S because the style guide specifies British English)
Otherwise LGTM. Thanks!